dask/dataframe/io/parquet/arrow.py

```diff
@@ -423,7 +218,8 @@ def _read_table_from_path(
         else:
             return tables[0]
     else:
-        with fs.open(path, mode="rb") as fil:
+        cache_type = "none" if columns or row_groups != [None] else "all"
```

Comment:
Yeah - I'm glad you caught this.  I can roll back this change.  The original purpose was dask-cudf related (where we definitely do want `cache_type="none"` for non-local filesystems). I agree that this may not always be the best general approach.

Reply:
Note that this change was related to [this comment](https://github.com/dask/dask/pull/8072#discussion_r707875495) (and the discussion in that thread). That is, I have dealt with a few read_parquet applications where `cache_type="none"` provides a clear performance bumb.  With that said, I think it would be best to address that issue in the follow-up PR where we integrate the cat_ranges optimization.